sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -42,6 +42,10 @@ import org.apache.spark.unsafe.types.UTF8String
     Examples:
       > SELECT _FUNC_('{"a":"b"}', '$.a');
        b
+      > SELECT _FUNC_('[{"a":"b"},{"a":"c"}]', '$[0].a');
```

Comment:
Please format the JSON of the example to make it look more pretty and formal, eg:
> SELECT _FUNC_('[{"a": "b"}, {"a": "c"}]', '$[0].a');